packages/ui/test/proposals/modals/AddNewProposalModal.test.tsx

```diff
@@ -1133,17 +1133,17 @@ describe('UI: AddNewProposalModal', () => {
           expect(await getCreateButton()).toBeDisabled()
         })
 
-        it('Invalid - group selected, amount not filled', async () => {
+        it('Invalid - group selected, amount automatically filled', async () => {
           await SpecificParameters.UpdateWorkingGroupBudget.selectGroup('Forum')
           await waitFor(() => expect(screen.queryByText(/Current budget for Forum Working Group is /i)).not.toBeNull())
 
-          expect(await getCreateButton()).toBeDisabled()
+          expect(await getCreateButton()).not.toBeDisabled()
```

Comment:
I think we should keep a real invalid case e.g: `Invalid - group selected, amount smaller than current stake filled`